src/OpenTelemetry.AutoInstrumentation/.publicApi/netcoreapp3.1/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,158 @@
+const OpenTelemetry.AutoInstrumentation.DuckTyping.DuckAttribute.DefaultFlags = System.Reflection.BindingFlags.Instance | System.Reflection.BindingFlags.Static | System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.FlattenHierarchy -> System.Reflection.BindingFlags
```

Comment:
Quick visual says it might need resorting?

Reply:
Whole file is autogenerated by VisualStudio quick fix :)
The same way was recommended on the .NET OTel contrib side.